src/scss/formapp/print.scss

```diff
@@ -1,10 +0,0 @@
-@media print {
```

Comment:
This needs to stay.  It's used by the PDF generator which prints.  Could maybe add a comment reflecting that.

Reply:
Oops, good call!

Added that file back with the comment.